addons/fix-editor-comments/addon.json

```
@@ -38,30 +38,35 @@
     {
       "name": "Fix dragging blocks with comments",
       "id": "fix-drag",
+      "description": "Fixes a bug where if you move a block with a comment attached to it, the comment will go back to where it was before once you switch sprites or reload the project.",
       "type": "boolean",
       "default": true
     },
     {
-      "name": "Add scroll bars to clipped comments",
+      "name": "Add scroll bars",
       "id": "scroll",
+      "description": "Adds scroll bars to long comments and allows you to scroll through them with the scroll wheel.",
       "type": "boolean",
       "default": true
     },
     {
       "name": "Directly edit standalone comments",
```

I think this setting might be better off called just:
```suggestion
      "name": "Directly edit comments",
```
...as long as it is explained in the setting description exactly what comments this applies to.